Layers/Renderers/Building/BuildingRenderer.gd

```diff
@@ -62,9 +71,19 @@ enum flag {
 
 func _ready():
 	_create_and_set_texture_arrays()
+	_find_roof_addon_ids()
 	super._ready()
 
 
+func _find_roof_addon_ids():
```

Comment:
Remove this and replace it with per-instance calls to Geodot's new `GeoFeatureLayer.get_features_by_attribute_filter`. So in the building instance, it will be something like:

```gdscript
addons = addon_layer.get_features_by_attribute_filter("build_id = %s" % [feature.get_id()])
```

Reply:
[171f19e](https://github.com/boku-ilen/landscapelab/pull/338/commits/171f19ef5f214b2d0896bc8c395de26afd292080)